README.md

```diff
@@ -86,15 +86,15 @@ and the data files in `/tmp/share`.
 
  ## Playing the game
  
- The game takes place on a rectangular areana where a snake continuously
+ The game takes place on a rectangular arena where a snake continuously
```

Comment:
I *think* this "on" should be "in"

```suggestion
 The game takes place in a rectangular arena where a snake continuously
```

Reply:
Yeah the snake is technically **insine** the arena so I agree with "in"